rq_dashboard/cli.py

```diff
@@ -30,7 +31,34 @@ def basic_http_auth(*args, **kwargs):
                 {'WWW-Authenticate': 'Basic realm="{}"'.format(realm)})
 
 
-def make_flask_app(config, username, password, url_prefix):
+LEGACY_CONFIG_OPTIONS = {
+    'REDIS_URL': 'RQ_DASHBOARD_REDIS_URL',
+    'REDIS_HOST': 'RQ_DASHBOARD_REDIS_HOST',
+    'REDIS_PORT': 'RQ_DASHBOARD_REDIS_PORT',
+    'REDIS_PASSWORD': 'RQ_DASHBOARD_REDIS_PASSWORD',
+    'REDIS_DB': 'RQ_DASHBOARD_REDIS_DB',
+    'REDIS_SENTINELS': 'RQ_DASHBOARD_REDIS_SENTINELS',
+    'REDIS_MASTER_NAME': 'RQ_DASHBOARD_REDIS_MASTER_NAME',
+    'RQ_POLL_INTERVAL': 'RQ_DASHBOARD_POLL_INTERVAL',
+    'WEB_BACKGROUND': 'RQ_DASHBOARD_WEB_BACKGROUND',
+    'DELETE_JOBS': 'RQ_DASHBOARD_DELETE_JOBS',
+}
+
+warning_template = "Configuration option {old_name} is depricated and will be removed in future versions. "\
```

Comment:
Typo; should read "deprecated".

Reply:
I left typos as is for now. Feel free to submit a p/r with these corrections.
Anyway I'm planning big redesign and hit them all with a hammer during It anyways